vllm/model_executor/layers/quantization/utils/marlin_utils.py

```diff
@@ -45,14 +45,21 @@ def query_marlin_supported_quant_types(
     if device_capability < 80:
         return []
 
+    if has_zp is None:
```

Comment:
It isn't clear what is the difference between has_zp being True, False, or None

Reply:
- has_zp is True: return quant_types that has zero points
- has_zp is False: return quant_types that has not zero points
- has_zp is None: both

The comment is added to the code.